scripts/post-update.sh

```diff
@@ -235,6 +235,51 @@ if ! git remote | grep -q local; then
     git remote add local ~/companion
 fi
 
+#Install isc-dhcp-server
+cd $HOME/companion
+echo "Checking for isc-dhcp-server..."
+while true
+do
+    if dpkg-query -W -f='${Status} ${Version}' isc-dhcp-server | grep 4.3; then
+        echo "Already installed"
+        sudo dpkg --configure -a
+        break
+    else
+        echo "Installing isc-dhcp-server"
+        sudo apt-get update --yes
+        sudo apt-get install isc-dhcp-server=4.3.* --yes
+        sudo dpkg --configure -a
+        echo "isc-dhcp-server installed"
+    fi
+done
+
+#Delete ip address if present from /boot/cmdline.txt
+# e.g. sed command removes any ip address with any combination of digits [0-9] between decimal points
+sudo sed -i -e 's/\s*ip=[0-9]*\.[0-9]*\.[0-9]*\.[0-9]*//' /boot/cmdline.txt
+
+#Copy network configuration files from Companion directory to respective configuration directories
+sudo cp /home/pi/companion/eth0-config /etc/network/interfaces.d/
+sudo cp /home/pi/companion/interfaces /etc/network/
+cp /home/pi/companion/dhcp-server-conf /home/pi
+cp /home/pi/companion/dhcp-server-defaults /home/pi 
+
+#Remove old configuration information if exists from default file
+sudo sed -i '/INTERFACES=/d' /etc/default/isc-dhcp-server
+sudo sed -i '/home\/pi\/dhcp/d' /etc/default/isc-dhcp-server
+
+#Remove old configuration information from dhcpd.conf file if exist
+sudo sed -i '/include \"\/home\/pi/d' /etc/dhcp/dhcpd.conf 
```

Comment:
if you run post update twice, then it adds the same two lines again and then the server start up fails. I checked it. thats why i am removing it first if it exist. otherwise it will keep adding to it everytime someone runs post-update. I can remove it. i just feel its safe to include that line.

Reply:
The line of code that appends the line to the file should only run one time, not every time post update runs (only the first time after we release this). See my comment in this file above.